digital_image_processing/change_contrast.py

```diff
@@ -0,0 +1,35 @@
+"""
+Changing contrast with PIL
+
+This algorithm is used in
+https://noivce.pythonanywhere.com/ python web app.

```

Comment:
Please provide a URL to the code repository as well.  We need to ensure that the code was published under a license similar to [__this repo's license__](https://github.com/TheAlgorithms/Python/blob/master/LICENSE.md).

Reply:
I wrote that. 
This is a hello world app I wrote with Flask https://noivce.pythonanywhere.com/ .

Reply:
Good.  Spelling: "novice"

Reply:
I am using free version. So novice isn't available.